### PR TITLE
Fix selection binding: add IronSelectableBehavior

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "paper-tabs",
-  "version": "0.9.1",
+  "version": "0.9.2",
```
Comment (cdata) [inline bower.json]: Thanks for bumping this :100:

File: paper-tabs.html [hunk 2]
```diff
@@ -200,7 +200,8 @@
     is: 'paper-tabs',
 
     behaviors: [
-      Polymer.IronResizableBehavior
+      Polymer.IronResizableBehavior,
+      Polymer.IronSelectableBehavior,
```
Comment (cdata) [inline paper-tabs.html]: I don't see how this could have been a problem :black_joker:

**Comment by cdata** [comment]
Good catch :+1:

**Comment by cdata** [comment]
Sorry, I had to revoke my emoji. Please give me a minute.

File: paper-tabs.html (hunk 2 above)
Comment (cdata) [inline paper-tabs.html]: Elements team did not end up agreeing with trailing comma on lists of things. Our convention is to omit the comma.

**Comment by cdata** [comment]
So, first: making the element apply `selectable` is redundant when we have the `iron-selector` in there. Only one of the two things should be required to get the desired effect.

The reason why it sort of seems to work without applying `selectable` is because the binding system creates accessors for `selected` on the tabs, even though tabs doesn't have `selected` in its `properties.

**Comment by cdata** [comment]
It seems like what we want is to apply `selectable` and remove the `iron-selector` element. Alternatively, we could mirror the `selected` property in the `properties` block for `paper-tabs`, and tell it to notify.

**Comment by justinfagnani** [comment]
Ok, I update the change to just add a notify `selected` property. PTAL

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**